### PR TITLE
News & Events main page.

### PR DESCRIPTION
**Pull request by hectoruch**
- Create URL on drupal admin site.
- Create a template for news&events URL.
- Create functions file.
- Add select filters and cover event block.

### CODE REVIEW

**Comment by hectoruch** [comment]
- Add events tiles.
- Add styles to event tiles.
- Add functions to put event tiles on news&events main page.

image

**Comment by hectoruch** [comment]
- Add box for load more events (only styles)

image

**Comment by hectoruch** [comment]
- Create endpoint to news.
- Functions to show news grid on the page.
- Functions to create pagination.

image

**Comment by hectoruch** [comment]
- Add loader on events grid.
- Add loaders on news grid.

File: sites/all/themes/custom/ogp_theme/lib/components/tile.js [hunk 1]
```diff
@@ -75,6 +75,26 @@ function appendSmallTiles(data, topContainer, gridNum, customClass) {
...
 
+function appendTilesEvent(data, container) {
+  if (data.length > 0) {
+    let html = '';
+    data.forEach((item) => {
+      html += `
+        <div class="small-12 medium-3 event-tile-container">
+          <div>
+            <h3 class="text -title-x-small">${moment(item.date.value).format('MMMM, DD')}</h3>
+            <span class="text language-text">Language: English</span>
```
Comment (edbrett) [inline sites/all/themes/custom/ogp_theme/lib/components/tile.js]: Should this not come from the field language we created yesterday?

**Pull request merged**